packages/frontend/src/stdlib/theories.tsx

```diff
@@ -20,6 +20,7 @@ stdTheories.add(
         id: "simple-olog",
         name: "Olog",
         description: "Ontology log, a simple conceptual model",
+        divisionCategory: "Data and Knowledge",
```

Comment:
I know I said "Data and knowledge," but I think I prefer "Knowledge and data" instead. Sorry.